src/status_im/contexts/chat/messenger/messages/view.cljs

```diff
@@ -19,57 +19,50 @@
     [utils.re-frame :as rf]))
 
 (defn- footer
-  [props]
+  [layout-height]
   (let [current-chat-id       (rf/sub [:chats/current-chat-id])
         able-to-send-message? (rf/sub [:chats/able-to-send-message?])]
     (when-not (string/blank? current-chat-id)
       (if able-to-send-message?
-        [composer/view props]
+        [composer/view layout-height]
         [contact-requests.bottom-drawer/view {:contact-id current-chat-id}]))))
 
 (defn- chat-screen
-  [{:keys [insets] :as props}]
+  [on-layout-done?]
   (let [theme                    (quo.theme/use-theme)
+        layout-height            (rn/use-ref-atom 0)
+        distance-from-list-top   (reanimated/use-shared-value 0)
+        chat-list-scroll-y       (reanimated/use-shared-value 0)
         alert-banners-top-margin (rf/sub [:alert-banners/top-margin])
-        chat-exist?              (rf/sub [:chats/current-chat-exist?])]
-    (when chat-exist?
-      [rn/keyboard-avoiding-view
-       {:style                    (style/keyboard-avoiding-container theme)
-        :keyboard-vertical-offset (- (if platform/ios? alert-banners-top-margin 0) (:bottom insets))}
-       [:<>
-        [list.view/messages-list-content props]
-        [scroll-to-bottom/button props]]
-       [messages.navigation/view props]
-       [footer props]])))
-
-(defn lazy-chat-screen
-  [chat-screen-layout-calculations-complete? *screen-loaded?*]
-  (let [screen-loaded?         (rf/sub [:shell/chat-screen-loaded?])
-        distance-from-list-top (reanimated/use-shared-value 0)
-        chat-list-scroll-y     (reanimated/use-shared-value 0)
-        props                  {:insets (safe-area/get-insets)
-                                :content-height (atom 0)
-                                :layout-height (atom 0)
-                                :distance-atom (atom 0)
-                                :distance-from-list-top distance-from-list-top
-                                :chat-list-scroll-y chat-list-scroll-y
-                                :chat-screen-layout-calculations-complete?
-                                chat-screen-layout-calculations-complete?}]
```

Comment:
These `atom`s are not created in a form-2 component, I moved the to be `use-ref-atom`.